modules/relocation-of-openshift-jenkins-images.adoc

```diff
@@ -0,0 +1,50 @@
+
+About the new image stream tags:
+
+`ocp-upgrade-redeploy`:: To update your Jenkins image when you upgrade {product-title}, you use this image stream tag in your Jenkins deployment configuration. This image stream tag corresponds to the existing `2` image stream tag of the `jenkins` image stream and the `latest` image stream tag of the `jenkins-agent-base` image stream. It employs an image tag specific to only one SHA or image digest. When changes of note are made to the `ocp-tools-4` image, such as for Jenkins security advisories, Red Hat Engineering updates the Cluster Samples Operator payload.
+
+`user-maintained-upgrade-redeploy`:: To manually redeploy Jenkins after you upgrade {product-title}, you use this image stream tag in your Jenkins deployment configuration. This image stream tag uses the least specific image version indicator available. When you redeploy Jenkins, you run the following command: `$ oc import-image jenkins:user-maintained-upgrade-redeploy -n openshift`. When you issue this command, the {product-title} `ImageStream` controller accesses the `registry.redhat.io` image registry and stores any updated images in the {product-title} internal image registry's slot for that Jenkins `ImageStreamTag` object. Otherwise, if you do not run this command, your Jenkins deployment configuration does not trigger a redeployment.
+
+`scheduled-upgrade-redeploy`:: To automatically redeploy the latest version of the Jenkins image when it is released, you use this image stream tag in your Jenkins deployment configuration. This image stream tag uses the periodic importing of image stream tags feature of the {product-title} image stream controller, which checks for changes in the backing image. If the image changes, for example, when we push changes due to a recent Jenkins security advisory, {product-title} triggers a redeployment of your Jenkins deployment configuration. See "Configuring periodic importing of image stream tags" in the following "Additional resources."
+
+
+.What happens with the `jenkins-agent-maven` and `jenkins-agent-nodejs` image streams in the `openshift` namespace?
+
+As mentioned, the associated in-payload `OpenShift Jenkins Maven` and `NodeJS Agent` images for {product-title} were deprecated in 4.10, and are removed from the {product-title} install payload in 4.11. They do not have alternatives defined in the `ocp-tools-4` repository. However, you can work around this by using the sidecar pattern described in the "Jenkins agent" topic mentioned in the following "Additional resources" section.
```

Comment:
Minimalism: 

`The associated payload `OpenShift Jenkins Maven` and `NodeJS Agent` images for {product-title} were deprecated in 4.10, and are removed from the {product-title} install payload in 4.11.`

Reply:
Updated. Thanks.